### DISCUSSION TITLE
Welcome to matRad Discussions!

### DISCUSSION THREAD

**Discussion by wahln** [e0404/matRad#478] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other researchers who would like to use or contribute to matRad. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas on enhancements or changes.
  * Engage with other community members.
  * Welcome others and are open-minded!
  * Introduce your projects & discuss if they should be incorporated into matRad

**Comment by markbangert** [comment]
Just wanted to say that matRad is awesome :) Thank you guys and keep up the good work...
